.dockerignore

```diff
@@ -1,15 +1,64 @@
+# vim:set ft=gitignore:
+
+# Git / Editor / Coverage / Linter
+.git
+.gitignore
+.dockerignore
+.idea
+*.swo
+*.swp
+html/
+htmlcov/
+.coverage
+.coverage.*
+.coveragerc
+coverage.xml
+*,cover
+cover/
+.pytest_cache/
+.tox/
+.cache
+nosetests.xml
+.mypy_cache/
+
+# CI
+.appveyor.yml
+.bumpversion.cfg
+.circleci
+.pyup.yml
+
+# Synapse
+docs/
+CHANGELOG.md
+Dockerfile
+MANIFEST.in
+synapse/tests
```

Comment:
We want to include the test code since its now part of the synapse package itself

Reply:
Re-added